inspire_utils/name.py

```diff
@@ -79,15 +79,20 @@ def __init__(self, name, constants=None):
         """
         if not constants:
             constants = ParsedName.constants
+        if without_titles:
+            constants.titles.remove(*constants.titles)
```

Comment:
why not 
`constants.titles = []`?

Reply:
I figured I'd use the nameparsers method for it since it was also used in the ```_prepare_nameparser_constants()``` when preparing the constants and I'm handling nameparsers variable. But I can also use that.

Reply:
I see, my point is that you need to remove every element from the list one by one in remove here which doesn't make sense for me if you can just assign empty list instead

Reply:
That's very true. I'll assign an empty list 👍